### ISSUE TITLE
VoiceOver: Unlabeled buttons in Toot app 1.10

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#363]
With Toot v1.10 having gained so much accessibility, there are only a few places where buttons are still missing accessible labels. Some derive a somewhat cryptic label from the images that are the button "labels", some others are not labeled at all. The below list contains what VoiceOver says, approximately. The places where I have found such buttons are:

1. When composing a toot:
   * "Circled plus library"
   * "Circled plus camera"
   * "circled plus poll"
   * When replying to a toot, an unlabeled button at the top left, below the Close button.
2. In a user profile screen:
   * Two unlabeled buttons below the "You follow them" text, roughly above the Mention and Message buttons.
3. In the Servers screen:
   * "Circled plus Add Server".
4. When viewing a toot/thread:
   * The button for the author's profile.
   * A second button before the Reply button, which sometimes opens the same profile, but sometimes the author of the first reply in the thread.
   * If the toot is a poll, each poll option is unlabeled.
   * If the poll is by oneself, the buttons/poll items and their percentages are also not labeled/read.

**Comment by MarcoZehe** [comment]
@DagAgren  Actually, the toot screen label is proper, should be re-added. I just updated the original description with items in the toot screen that are unlabeled, see numbered item 4 and its sub items.

**Comment by MarcoZehe** [comment]
These are all still missing in the current 1.12 release.

**Comment by DagAgren** [comment]
Could I set you up with a Testflight for trying the fixes for these?

**Comment by pitermach** [comment]
Sure, I'd be happy to test any potential fixes.

**Comment by DagAgren** [comment]
Finally think I've managed to sort all of these out, and some.
